librz/util/compression.c

```diff
@@ -112,16 +113,6 @@ RZ_API ut8 *rz_inflatew(RZ_NONNULL const ut8 *src, int srcLen, int *srcConsumed,
 	return NULL;
 }
 
-/**
- * \brief deflate uncompressed data to zlib or gzipped, use MAX_WBITS as the window size logarithm.
- * \see rz_deflatew()
- */
-RZ_API ut8 *rz_deflate(RZ_NONNULL const ut8 *src, int srcLen, int *srcConsumed, int *dstLen) {
```

Comment:
Why those are removed?

Reply:
they are not removed, just moved to #if/#else so that i can compile rz-util easily without needing zlib (in case of cross compiiling)